server/rest/src/test/java/org/infinispan/rest/search/RestHitCountAccuracyTest.java

```diff
@@ -111,7 +111,7 @@ public void test() throws Exception {
       assertEquals(body.at("hit_count").asInteger(), ENTRIES);
    }
 
-   private static void writeEntries(int entries, RestCacheClient cacheClient) {
+   private void writeEntries(int entries, RestCacheClient cacheClient) {
```

Comment:
Any reason this was changed? We should prefer static if it can be.

Reply:
It's a test. Also I did this temporarily to invoke `AbstractIinfinispanTest.checkFDs()` in multiple places